src/schedule/zephyr_dma_domain.c

```diff
@@ -56,6 +56,27 @@ struct zephyr_dma_domain_data {
 	 * associated channel was triggered.
 	 */
 	bool triggered;
+	/* used to keep track of how many pipeline tasks point
+	 * to the same data structure.
+	 */
+	int usage_count;
+	/* used to keep track of how many times data->triggered
```

Comment:
can we similarly simplify this and remove "used to keep track of?"

Reply:
Sure